bucket/veusz.json

```diff
@@ -9,7 +9,7 @@
             "hash": "7e47e9d08e9393ec1a8d786716d5e11cc93ac7a10a8164e7490736c7a2960dc4"
         }
     },
-    "pre_install": "Remove-Item \"$dir\\`$*\" \"$dir\\uninst*\" -Force -Recurse",
+    "pre_install": "Remove-Item @(\"$dir\\`$*\", \"$dir\\uninst*\") -Force -Recurse",
```

Comment:
```suggestion
    "pre_install": "Remove-Item \"$dir\\`$*\",\"$dir\\uninst*\" -Force -Recurse",
```

Reply:
The array is necessary, or else the Remove-Item command will not work properly.

Reply:
the original line not compatible with powershell 5.1, but separating them with comma is works, tested on powershell 5.1/core LTS 7.2.6.

Reply:
oh ok, I'll change it.